samples/04_gis_analysts_data_scientists/covid19_part2_timeseries_analysis.ipynb

```diff
@@ -3680,7 +3680,7 @@
     "\n",
```

Comment:
<p>Hi Dave - The link to StoryMap in [here] looks good but the link to 'this notebook' looks weird, it is showing me "https://app.reviewnb.com/Esri/arcgis-python-api/pull/819/files/covid19_part3_predictive_analysis.ipynb". </p><p> </p><p>Is this how a link to sample notebook would look like in ReviewNB and if not could you please fix the link.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/819/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='91'/>

Reply:
Hi Mohit that is expected. ReviewNB isn't aware of other notebooks in the local directory, so you can ignore any invalid links when reviewing in reviewNB. Invalid links when you run the notebook locally are what we are concerned about